app/routes/datasets/queries.py

```diff
@@ -646,25 +646,7 @@ def _get_default_layer(dataset, pixel_meaning):
 
 async def _get_data_environment(grid: Grid) -> DataEnvironment:
     # get all Raster tile set assets
-    latest_tile_sets = await (
-        AssetORM.join(VersionORM)
-        .select()
-        .with_only_columns(
-            [
-                AssetORM.dataset,
-                AssetORM.version,
-                AssetORM.creation_options,
-                AssetORM.asset_uri,
-                AssetORM.metadata,
-            ]
-        )
-        .where(
-            and_(
-                AssetORM.asset_type == AssetType.raster_tile_set,
-                VersionORM.is_latest == True,  # noqa: E712
-            )
-        )
-    ).gino.all()
+    latest_tile_sets = await assets.get_raster_tile_sets()
```

Comment:
did you test OTF at all on staging?

Reply:
good call, I just did on a couple of geostores.. here is [one](staging-data-api.globalforestwatch.org/dataset/umd_glad_sentinel2_alerts/v20230202/query/json?geostore_id=46df30d9a44192efd90923af22e4fd0e&geostore_origin=rw&sql=select+latitude,+longitude,+umd_glad_sentinel2_alerts__date,+umd_glad_sentinel2_alerts__confidence+from+umd_glad_sentinel2_alerts+where+umd_glad_sentinel2_alerts__date+%3E+'2022-10-22'+and+umd_glad_sentinel2_alerts__date+%3C+'2023-04-20'+ORDER+BY+umd_glad_sentinel2_alerts__date+DESC+LIMIT+10000).